Upgrading Spring brings in a different version of `micrometer-observation` in Spring Example
```
[ERROR] Dependency convergence error for io.micrometer:micrometer-observation:jar:1.15.1. Paths to dependency are:
[ERROR] +-org.glassfish.jersey.validation:dependency-convergences:war:0.0.1
[ERROR]   +-org.glassfish.jersey.ext:jersey-micrometer:jar:3.1.99-SNAPSHOT:compile
[ERROR]     +-io.micrometer:micrometer-core:jar:1.15.1:compile
[ERROR]       +-io.micrometer:micrometer-observation:jar:1.15.1:compile
[ERROR] and
[ERROR] +-org.glassfish.jersey.validation:dependency-convergences:war:0.0.1
[ERROR]   +-org.glassfish.jersey.examples:helloworld-spring-webapp:jar:3.1.99-SNAPSHOT:compile
[ERROR]     +-org.springframework:spring-context:jar:6.2.11:compile
[ERROR]       +-io.micrometer:micrometer-observation:jar:1.14.11:compile
[ERROR] -> [Help 1]
```

--- a/examples/helloworld-spring-webapp/pom.xml
+++ b/examples/helloworld-spring-webapp/pom.xml
@@ -76,6 +76,12 @@
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-context</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.micrometer</groupId>
+                    <artifactId>micrometer-observation</artifactId>
+                </exclusion>
+            </exclusions>
             <version>${spring6.version}</version>
         </dependency>
         <dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -2078,7 +2078,11 @@
                 <artifactId>opentracing-util</artifactId>
                 <version>${opentracing.version}</version>
             </dependency>
-
+            <dependency>
+                <groupId>io.micrometer</groupId>
+                <artifactId>micrometer-observation</artifactId>
+                <version>${micrometer.version}</version>
+            </dependency>
         </dependencies>
     </dependencyManagement>
 

--- a/pom.xml
+++ b/pom.xml
@@ -2078,11 +2078,6 @@
                 <artifactId>opentracing-util</artifactId>
                 <version>${opentracing.version}</version>
             </dependency>
-            <dependency>
-                <groupId>io.micrometer</groupId>
-                <artifactId>micrometer-observation</artifactId>
-                <version>${micrometer.version}</version>
-            </dependency>
         </dependencies>
     </dependencyManagement>
 

